modules/instance_template/main.tf

```diff
@@ -71,13 +71,13 @@ resource "google_compute_instance_template" "tpl" {
       boot         = lookup(disk.value, "boot", null)
       device_name  = lookup(disk.value, "device_name", null)
       disk_name    = lookup(disk.value, "disk_name", null)
-      disk_size_gb = lookup(disk.value, "disk_size_gb", null)
+      disk_size_gb = lookup(disk.value, "disk_size_gb", lookup(disk.value, "disk_type", null) == "local-ssd" ? "375" : null)
```

Comment:
We shouldn't add a default here.

Reply:
What do you mean? Set it as a `local` and use that instead? (Because scratch devices need to be exactly 375Gb)

Reply:
Hi @morgante, I'm wondering if you have any comments about this so I can push the change to get it merged.

Thanks!

Reply:
Got it, I think this is fine as a default then.